src/libertem/io/dataset/tvips.py

```diff
@@ -75,6 +79,37 @@ def convert_to_python(self, raw_data):
 ]
 
 
+image_header_v2_dtype = [
+    ('ICounter', 'u4'),                 # image counter, continues through all files
+    ('ITime', 'u4'),                    # unix time stamp
+    ('IMS', 'u4'),                      # timestamp milliseconds
+    ('LUT_Index', 'u4'),                # LUT index (?)
+    ('Faraday', 'float32'),             # faraday cup value (unit?)
+    ('TEM_Mag', 'u4'),                  # magnification (unit?)
```

Comment:
the pixel size of acquired image can most probably be calculated from real pixel size of the camera: `px_size = px_size_real * binning / TEM_Mag`
`px_size_real` is 15.5 um for [XF416(R)](https://www.tvips.com/camera-systems/temcam-xf-series/)